tools/assemblystats/assembly_stats_txt.xml

```diff
@@ -1,31 +1,34 @@
-<tool id="assemblystats" name="assemblystats" version="1.0.1">
+<tool id="assemblystats" name="assemblystats" version="1.1.0">
 	<description>Summarise an assembly (e.g. N50 metrics)</description>
 	<requirements>
 		<requirement type="package" version="1.6.924">perl-bioperl</requirement>
-		<requirement type="package" version="5.0.4">gnuplot</requirement>
+		<requirement type="package" version="5.2.7">gnuplot</requirement>
```

Comment:
Should also specify python 3 here as a dependency

Reply:
Good idea, added!